app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/PublicationsFragment.java

```diff
@@ -39,6 +102,37 @@ public void initializeUI() {
     @Override
     public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         inflater.inflate(R.menu.menu, menu);
+
+        final MenuItem item = menu.findItem(R.id.menu_search);
+        final SearchView searchView = (SearchView) MenuItemCompat.getActionView(item);
+        searchView.setQueryHint("Search by Title");
+        searchView.setOnQueryTextListener(this);
+        searchView.setOnCloseListener(new SearchView.OnCloseListener() {
+            @Override
+            public boolean onClose() {
+                return false;
+            }
+        });
+
         super.onCreateOptionsMenu(menu, inflater);
     }
-}
+
+    @Override
+    public boolean onQueryTextSubmit(String query) {
+        return false;
+    }
+
+    @Override
+    public boolean onQueryTextChange(String query) {
+        if (query.length() <= 0) {
```

Comment:
You may wonder why I'm not saying you should extract this as well. This is a clear comparison and it is, in itself descriptive. i.e you're comparing the length to a given number. But below, you're saying scroll to position 0. Really, what is position 0 ? The default position. So extract it as a constant and name it as such :)